client/src/pages/public/ConfirmInterest.js

```diff
@@ -62,7 +62,7 @@ export default (props) => {
   return (
     <Page hideEmployers={true}>
       <Card className={classes.root}>
-        <Box display='flex' flexDirection='column' height='100%' justifyContent='center'>
+        <Box display='flex' flexDirection='column'>
```

Comment:
Was this intentional?

Reply:
This was from the merge, I'll re-add it.